vaadin-testbench-core-junit5/src/test/java/com/vaadin/testbench/browser/BrowserHubTest.java

```diff
@@ -43,4 +45,25 @@ public void hubFromAnnotationOrSystemProperty() {
         }
     }
 
+    @Test
+    public void hubPortFromDefaultValueOrParametersSetter() {
+        String oldProperty = System.getProperty(HUB_PORT_PROPERTY);
+        try {
+            // Default must be the "official" 4444 port for backwards compatibility
+            Assertions.assertEquals(getExpectedHubUrl(4444), getHubURL(getClass()));
```

Comment:
This assertions fail in the CI environment because `getHubURL()` returns the pre-configured sauce lab URL.
We should either change the assertion or add a workaround to temporarily skip the sauce lab integration

```java
    private String sauceUser;

    @BeforeEach
    void temporaryDisableSauceLab() {
        if (SauceLabsIntegration.isConfiguredForSauceLabs()) {
            sauceUser = System.getProperty("sauce.user");
            System.clearProperty("sauce.user");
        }
    }

    @AfterEach
    void restoreSauceLab() {
        if (SauceLabsIntegration.isConfiguredForSauceLabs()
                && sauceUser != null) {
            System.setProperty("sauce.user", sauceUser);
        }
    }
```

Reply:
Yeah, I've seen that yesterday and it should be fixed in the last commit here: https://github.com/vaadin/testbench/pull/1658/commits/cedc8e24de8b60d067a276e19749a3ff69701fba
I simply added a call to the getHubHostname method, which should provide me with a correct URL.

Reply:
Oh interesting, I thought that the getHubHostname call would fix it, but it returns the IP which does not match the actual hostname used. I'll change the approach for the assertion than.

Reply:
I'm using the same mechanic like in the other test now
```java
  // Reset the hub hostname property (removes the saucelab url in CI)
  System.clearProperty(HUB_HOSTNAME_PROPERTY);
```

I'm not a fan of the try finally which resets the properties tho. If it is fine for you I would move it from both tests into @BeforeEach and @AfterEach

Reply:
I suppose that will not be enough. `getHubUrl()` will still return the SauceLab url.
But you can test it locally with `mvn test -Dtest=BrowserHubTest -Dsauce.user=yyy -Dsauce.sauceAccessKey=xxx`, from the `vaadin-testbench-core-junit5` module.

> I'm not a fan of the try finally which resets the properties tho. If it is fine for you I would move it from both tests into `@BeforeEach` and `@AfterEach`

It looks like a good improvement. Please go ahead